frontend/src/components/Organization.tsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import * as api from '../api';
+import { User } from '../services/auth';
+import Errors from './Errors';
+import Loader from './Loader';
+
+interface OrgProps {
+  user: User;
+  orgName: string;
+}
+
+interface OrgState {
+  done: boolean;
+  org: api.OrgResponse | undefined;
+  errors: string[];
+
+  config: string;
+}
+
+class Organization extends React.Component<OrgProps, OrgState> {
+  constructor(props: OrgProps) {
+    super(props);
+
+    this.state = {
+      done: false,
+      org: undefined,
+      errors: [],
+      config: ''
+    };
+
+    this.handleConfigChange = this.handleConfigChange.bind(this);
+    this.handleConfigSave = this.handleConfigSave.bind(this);
+  }
+
+  public componentDidMount() {
+    return api
+      .org(this.props.orgName)
+      .then(resp =>
+        this.setState({
+          done: true,
+          org: resp,
+          errors: [],
+          config: resp.config
+        })
+      )
+      .catch(err => {
+        this.setState({
+          done: true,
+          org: undefined,
+          errors: err,
+          config: ''
+        });
+      });
+  }
+
+  public render() {
+    if (!this.state.done) {
+      return <Loader />;
+    }
+
+    const org = this.state.org;
```

Comment:
don't we have linter error here?

Reply:
No. I changed in 2f85ec7, I don't mind one way or the other.

Reply:
oh, no need to change. I meant eslint would complain here and ask to write it as `const {org} = this.state`. Good to know that tslint doesn't.